### PR TITLE
Desktop version: Fourth Project in my Portfolio

### PR DESCRIPTION
**Pull request by Tumworobere**
**In this milestone, I achieved the following:**
- I built a responsive desktop version of my portfolio
   - I created my-portfolio repository and cloned it into my local machine.
   - implemented linter, Github Flow, and professional documentation 
   - build a template from Figma by using CSS Flexbox and  media queries

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # My-Portfolio
-This is my second microverse project. In this project I built my portfolio(Mobile version).Thismarks the second step for the portfolio project.The main objective for this project was to build template from figma using CSS and Flexbox. I created my-portfolio reposotory and cloned into my local machine. I have implemented linter,Github Flow and professional documentation which is one of the project building requirements at microverse.
+This is my second project in the Microverse in which I built my portfolio . This is my responsive desktop version. The objective was to build template from figma by using CSS Flexbox. For that I created my-portfolio repo and cloned into my local machine.I have implemented linter,Github Flow and professional documentation which is one of the project building requirements at microverse.
```
Comment (SajjadAhmad14) [inline README.md:3]: Please add instructions to run the project locally.

File: style.css [hunk 2]
```diff
@@ -482,58 +460,254 @@ textarea::placeholder {
...
+  .work-container {
+    width: 991px;
+  }
+
+  .work-container h2.title::after {
+    width: 620px;
+  }
+}
+
+@media (min-width: 1200px) {
```
Comment (SajjadAhmad14) [inline style.css:698]: Please use value for min-width property should be in range of 992px - 1100px

File: index.html [hunk 3]
```diff
@@ -1,324 +1,210 @@
...
+        <a href="#"> Tumworobere Annah</a>
+        <nav class="nav-menu">
+            <ul class="menu-list"> 
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+                <li><img src="./images/mail.png" alt="mail icon"></li>
+            </ul>
+            <img class="icon" src="./images/Vector.png" alt="Vector image ">
+        </nav>
```
Comment (SajjadAhmad14) [inline index.html:21]: Please make nav links clickable.

File: index.html [hunk 4]
```diff
@@ -1,324 +1,210 @@
...
-          </p>
-          <div class="languages">
-            <ul>
-              <li>css</li>
-              <li>html</li>
-              <li class="bootstrap">bootstrap</li>
-              <li>Ruby</li>
-            </ul>
-          </div>
-          <button type="button" class="project-btn">See project</button>
```
Comment (SajjadAhmad14) [inline index.html:74]: Please add hover and active styles to buttons by following the design.

File: index.html [hunk 5]
```diff
@@ -1,324 +1,210 @@
...
-              sign-ups required. has been the industry's standard
-            </p>
-            <div class="languages">
-              <ul>
-                <li class="project-language">html</li>
-                <li class="project-language">bootstrap</li>
-                <li class="project-language">Ruby</li>
-              </ul>
-            </div>
-            <button type="button" class="project-btn">See project</button>
```
Comment (SajjadAhmad14) [inline index.html:90]: Please add hover and active styles to buttons by following the design.

File: index.html [hunk 6]
```diff
@@ -1,324 +1,210 @@
...
-              </ul>
+        <section class="container">
+            <div class="text">
+                <h1> Hey There. I'm Annah</h1>
+                <h3>I'm a Software Developer.</h3>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If
+                    you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
             </div>
-            <button type="button" class="project-btn">See project</button>
```
Comment (SajjadAhmad14) [inline index.html:108]: Please add hover and active styles to buttons by following the design.

File: index.html [hunk 7]
```diff
@@ -1,324 +1,210 @@
...
+            <div class="icon">
+                <ul>
+                    <li><a href="#"><img src="./images/img1.png" alt="Github-icon"></a></li>
+                    <li><a href="#"><img src="./images/img2.png" alt="LinkeIn-icon"></a></li>
+                    <li><a href="#"><img src="./images/img3.png" alt="AngelList-icon"></a></li>
+                    <li><a href="#"><img src="./images/img4.png" alt="Twitter-icon"></a></li>
+                    <li><a href="#"><img src="./images/img5.png" alt="Medium-icon"></a></li>
+                </ul>
             </div>
-            <button type="button" class="project-btn">See project</button>
```
Comment (SajjadAhmad14) [inline index.html:126]: Please add hover and active styles to buttons by following the design.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Tumworobere, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
PR has a title and summary. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 7 above)
Comment (Tumworobere) [reply index.html:126]: Thanks @SajjadAhmad14  i will work on the requested changes!

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         

## Thank you for implementing all the project requirements, you've done a great job :trophy:  . There is nothing else to say other than... it's time to merge it :champagne:  :tada: Congratulations

Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**